cmd/deploy/deploy.go

```diff
@@ -72,19 +72,19 @@ type Options struct {
 	ManifestPathFlag string
 	// ManifestPath is the path to the manifest used though the command execution.
 	// This might change its value during execution
-	ManifestPath     string
-	Name             string
-	Namespace        string
-	K8sContext       string
-	Variables        []string
-	ServicesToDeploy []string
-	Timeout          time.Duration
-	Build            bool
-	Dependencies     bool
-	RunWithoutBash   bool
-	RunInRemote      bool
-	Wait             bool
-	ShowCTA          bool
+	ManifestPath          string
+	Name                  string
+	Namespace             string
+	K8sContext            string
+	Variables             []string
+	StackServicesToDeploy []string
```

Comment:
why did you rename this to `StackServicesToDeploy`?

Reply:
`ServicesToDeploy` was used through different parts of the command and was confusing (at least for me) while validating the user input so I decided to make it more clear. Let me know if we should rename back to its name